src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = 0;
+  const baseRate = 40;
+  const sevenDaysOff = 50;
+  const threeDaysOff = 20;
+
+  if (days >= 7) {
+    totalCost = baseRate * days - sevenDaysOff;
+  } else if (days >= 3) {
+    totalCost = baseRate * days - threeDaysOff;
```

Comment:
Feel free to return right away if you've found the proper condition
```suggestion
    return baseRate * days - threeDaysOff;
```